### ISSUE TITLE
I am trying to visualize the results of an example I ran (sarsa.cartpole). The example ran fine and I got the results of the training, however, I can't visualize the results. Following the instructions given (from rl.callbacks import WandbLogger) raises an error [cannot import name 'WandbLogger']. The package (wandb) is installed and all the dependecies are installed. If this does not work, how else can I visualize the results and store the generated data (episode, reward, action,...etc)? 

### ISSUE THREAD

**Issue by malsaidi93** [keras-rl/keras-rl#369]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question in the [Keras-RL Google group](https://groups.google.com/forum/#!forum/keras-rl-users) or [join the Keras-RL Gitter channel](https://gitter.im/keras-rl/Lobby) and ask there instead of filing a GitHub issue.

Thank you!

- [x] Check that you are up-to-date with the master branch of Keras-RL. You can update with:
`pip install git+git://github.com/keras-rl/keras-rl.git --upgrade --no-deps`

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
`pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps`

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short). If you report an error, please include the error message and the backtrace.
